.github/workflows/ci.yml

```diff
@@ -43,6 +43,34 @@ jobs:
         file: ./coverage.txt
         flags: unittests
         name: codecov-umbrella
+  unit-test-arm:
+    runs-on: ubuntu-latest
+    name: unit-tests-qemu-arm
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install QEMU
```

Comment:
It may be a bit costly to use the test machine as a build machine over time. Have you thought of using e.g. 
https://github.com/docker/setup-qemu-action with a particular platform?